include/strf/detail/printers_tuple.hpp

```diff
@@ -45,7 +47,7 @@ class simple_tuple_impl<std::index_sequence<I...>, T...>
 
     template <typename ... Args>
     constexpr explicit simple_tuple_impl(simple_tuple_from_args, Args&& ... args)
-        : detail::indexed_obj<I, T>{args}...
+        : indexed_obj<I, T>(args)...
```

Comment:
It is necessary to also change `indexed_obj`, otherwise GCC fail to compile. 

```
template <std::size_t I, typename T>
struct indexed_obj
{
    explicit constexpr indexed_obj(const T& _obj)
        : obj(_obj)
    {
    }
    constexpr indexed_obj(const indexed_obj&) = default;
    constexpr indexed_obj(indexed_obj&&) = default;
    
    T obj;
};
```